.env

```diff
@@ -0,0 +1,5 @@
+DOCKER_APP_HOST_PORT=3852
```

Comment:
We should change default port, because the port 3852 is used in `e-motion-api` for ` escooters-redis-dev` container